docs/medical-api/more-info/api-tools.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Api Tools"
+description: "Overview of Metriport's API schemas, SDKs, and Postman collection"
+---
+
+## OpenAPI Specification
+
+Our OpenAPI specification defines the structure of our API, including endpoints, request/response formats, and available operations. You can find our OpenAPI definition in our GitHub repository:
+
+[Metriport OpenAPI Specification](https://github.com/metriport/metriport/tree/develop/fern/definition)
+
+## SDKs
+
+We provide Software Development Kits (SDKs) in multiple programming languages to make it easier for developers to integrate with our API. Our SDKs are available for the following languages:
+
+### Python
+
+- Package: [metriport](https://pypi.org/project/metriport/)
+- Installation: `pip install metriport`
```

Comment:
if we're going to add these to docs, we need to maintain them, updated the spec, and re-generate + publish

unless we do that, I'd suggest we only advertise our ts SDK in the docs, and tell people if they want to get SDks in other langs, to generate them off of the OpenAPI spec

Reply:
Gotcha we currently already do that so ill just leave as is then